### DISCUSSION TITLE
Why retrieving dashboards?

### DISCUSSION THREAD

**Discussion by chriss158** [BimmerGestalt/HassGestalt#2] [General]
I am trying to understand why you are reading the dashboards to get the entities? I could assume that this might be difficult depending on the structure of the dashboards.  At least this did not work for me with a simple dashboard structure. Only after I made a small code adjustment.

At the moment the entities are only lined up in a list. Wouldn't it then make more sense to select the entities you want to display in the app? 

Or do you have something else planned that you have to pull the dashboards?

**Comment by hufman** [comment]
I apologize for not seeing this sooner! It never sent a notification email to me, I sincerely apologize!
I don't know of a good way to group entities together besides dashboard definitions. The Fetch States API doesn't seem to return an entity's room, to implement the Overview list of all entities. So instead, I show a list of dashboards in the car to view, and I allow some to be favorited to show at the top level in the car app. This additionally leans into the different interaction mode of the car interface, and how someone in a car would probably only want to see a specific subset in the car instead of browsing the entire list of entities.

**Reply by Aleloberto** [reply]
Hi, i don't see any dashboard. I try to realize simple dashboard with only one entity..but it doesn't work.
Obviously, demo work fine.
I use nabu casa for external access..how can i solve?
Thank's a lot

**Reply by hufman** [reply]
Can you look in the output of `adb logcat -d` for HassWs messages? One of the first things the app does is to load a list of dashboards, and HassWs logs the response which should include the full list of dashboards.
HassGestalt only supports dashboards with `component_name:lovelace`. What does your dashboard result look like?

**Comment by Aleloberto** [comment]
Hi,
I send you 2 mail (necessary for attachments dimensions)
Output file as attached.

I've done this steps two times ((adb logcat > file.txt     and    adb logcat -d > file2.txt):
-Start adb logcat
- open HASS GESTALT app
-wait some time
-stop adb logcat

Thank's a lot

Ottenere Outlook per Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Aleloberto** [comment]
Here second mail with file2.txt

Thank's

Ottenere Outlook per Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by hufman** [reply]
Github email replies don't include attachments, please email me directly

**Comment by Aleloberto** [comment]
Hi, I send you 2 separate mails with attachment.

Thank's

Ottenere Outlook per Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by hufman** [reply]
Indeed you did! I was unable to determine any clues, however.
